src/test/regress/pg_regress_multi.pl

```diff
@@ -560,6 +577,20 @@ ()
     die "a file already exists at $mitmFifoPath, delete it before trying again";
   }
 
+  system("lsof -i :$mitmPort");
+  if (! $?) {
+    die "cannot start mitmproxy because a process already exists on port $mitmPort";
+  }
+
+  if ($Config{osname} eq "linux")
```

Comment:
How did this ever work in Windows? Does `netstat` do anything there, or am I missing something?

Reply:
nope, this doesn't work on Windows